policy-controller/core/src/routes.rs

```diff
@@ -91,6 +96,15 @@ pub enum QueryParamMatch {
     Regex(String, Regex),
 }
 
+pub trait GenericRouteMatch: Any + Default {
```

Comment:
I think this is only necessary to make the default inbound routes generic, which isn't something we want.  I think this can be removed.